pkg/networkmanager/interfaces.js

```diff
@@ -1722,6 +1722,19 @@ PageNetworking.prototype = {
             self.rx_plot.resize();
             self.tx_plot.resize();
         });
+
+        const desc = _("Network bonding is a method to combine or aggregate network interfaces to provide a logical interface with higher throughput or redundancy.");
```

Comment:
"combine or aggregate" seems a bit redundant to me. To shorten it a bit, maybe" A bond combines multiple network interfaces into one logical interface with higher throughput or redundancy."?

Reply:
I lifted this directly from the documentation. Also checked with Lucie, and she was OK with it.
Let me sync up with her and see if she has any concerns about the shorter string.

Reply:
I see. It's not important, feel free to ignore this.